src/Layout/index.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { gtaModel } from '../constants/gtaModel';
+import { GtaVersions } from '../Components';
+
+import '../Styles/styles.css';
```

Comment:
Writing all of the styles in one css file is wrong. Each compoennt should have its own.